tests/excerptexport/models/test_export_model.py

```diff
@@ -0,0 +1,31 @@
+from datetime import timedelta
+
+import pytest
+
+from django.utils.datetime_safe import datetime
+
+
+@pytest.mark.django_db
+def test_created_export_has_correct_time_stamp(export):
```

Comment:
Importing is not a good idea, as it would need a request object to be constructed, which is more finicky than having the same code copied over again.